### ISSUE TITLE
[GHOST] error on install/upgrade

### ISSUE THREAD

**Issue by jalberto** [bitnami/charts#11835]
### Name and Version

bitnami/ghost 19.0.16

### What steps will reproduce the bug?

1. helm upgrade bitnami/ghost
2.  {"errors":[{"message":"Setup has already been completed.","context":null,"type":"NoPermissionError","details":null,"property":null,"help":null,"code":null,"id":"919684b0-1f0d-11ed-b976-fdf77fc63ed5","ghostErrorCode":null}]}

### Are you using any custom parameters or values?

nothing related to PV

### What is the expected behavior?

to work

### What do you see instead?

 {"errors":[{"message":"Setup has already been completed.","context":null,"type":"NoPermissionError","details":null,"property":null,"help":null,"code":null,"id":"919684b0-1f0d-11ed-b976-fdf77fc63ed5","ghostErrorCode":null}]}

**Comment by jalberto** [comment]
I also tried to remove the `depoy` object (but not the mysql stateful one) then `upgrade` again to recreate the `deploy`, and get same error

**Comment by carrodher** [comment]
Please, [note that the Bitnami team is not actively monitoring GitHub Issues for this Helm chart](https://github.com/bitnami/charts/tree/master/bitnami/ghost#community-supported-solution). This issue will stay open for 20 days to allow the community to contribute, after 15 days without activity the issue will be marked as stale, being closed after 5 days.

If you would like to contribute to this Helm chart by fixing a bug or adding a new feature, feel free to create a PR, the Bitnami team will be happy to review it and provide feedback.

**Comment by jalberto** [comment]
Adding this doesn't fix the issue either:

```
volumePermissions:
  enabled: true
```

**Comment by jalberto** [comment]
I fully deleted the volume and did deploy again, everything recreated, same error

**Comment by jalberto** [comment]
when `ghostSkipInstall: true` I get this:

```
ghost 09:59:19.79 
ghost 09:59:19.79 Welcome to the Bitnami ghost container
ghost 09:59:19.80 Subscribe to project updates by watching https://github.com/bitnami/containers
ghost 09:59:19.80 Submit issues and feature requests at https://github.com/bitnami/containers/issues
ghost 09:59:19.80 
ghost 09:59:19.81 INFO  ==> Configuring libnss_wrapper
ghost 09:59:19.87 INFO  ==> Validating settings in MYSQL_CLIENT_* env vars
ghost 09:59:19.97 DEBUG ==> Validating settings in GHOST_* environment variables...
ghost 09:59:20.05 WARN  ==> You set the environment variable ALLOW_EMPTY_PASSWORD=yes. For safety reasons, do not use this flag in a production environment.
ghost 09:59:20.06 INFO  ==> Restoring persisted Ghost installation
ghost 09:59:20.16 INFO  ==> Trying to connect to the database server
ghost 09:59:20.17 DEBUG ==> Getting database.connection.host from Ghost configuration
ghost 09:59:20.35 DEBUG ==> Getting database.connection.port from Ghost configuration
ghost 09:59:20.47 DEBUG ==> Getting database.connection.database from Ghost configuration
ghost 09:59:20.64 DEBUG ==> Getting database.connection.user from Ghost configuration
ghost 09:59:20.75 DEBUG ==> Getting database.connection.password from Ghost configuration
ghost 09:59:20.87 DEBUG ==> Executing SQL command:
SELECT 1
1
ghost 09:59:20.95 INFO  ==> ** Ghost setup finished! **

ghost 09:59:20.98 INFO  ==> ** Starting Ghost **

Love open source? We’re hiring Node.js Engineers to work on Ghost full-time.
https://careers.ghost.org/product-engineer-node-js

- Inspecting operating system
[2022-08-19 09:59:27] INFO Ghost is running in production...
[2022-08-19 09:59:27] INFO Your site is now available on http://localhost:2368/
[2022-08-19 09:59:27] INFO Ctrl+C to shut down
[2022-08-19 09:59:27] INFO Ghost server started in 2.784s
events.js:377
      throw er; // Unhandled 'error' event
      ^

Error: EACCES: permission denied, open '/opt/bitnami/ghost/versions/5.7.1/content/logs/http___localhost_2368_production.error.log'
Emitted 'error' event on WriteStream instance at:
    at internal/fs/streams.js:335:14
    at FSReqCallback.oncomplete (fs.js:180:23) {
  errno: -13,
  code: 'EACCES',
  syscall: 'open',
  path: '/opt/bitnami/ghost/versions/5.7.1/content/logs/http___localhost_2368_production.error.log'
}
Stream closed EOF for ghost/ghost-798f979885-pf5hn (ghost)
```
